diff -ru common/lib/modules/fglrx/build_mod/firegl_public.c common/lib/modules/fglrx/build_mod.2.6.26-8-6/firegl_public.c
--- common/lib/modules/fglrx/build_mod/firegl_public.c
+++ common/lib/modules/fglrx/build_mod.2.6.26-8-6/firegl_public.c
@@ -4956,8 +4956,9 @@ static unsigned long kasSetExecutionLevel(unsigned long level)
 {
     unsigned long orig_level;
 
-    orig_level = __get_cpu_var(kasExecutionLevel);
+    orig_level = get_cpu_var(kasExecutionLevel);
     __get_cpu_var(kasExecutionLevel) = level;
+    put_cpu_var(kasExecutionLevel);
 
     return orig_level;
 }
@@ -4969,7 +4970,12 @@ static unsigned long kasSetExecutionLevel(unsigned long level)
  */
 static unsigned long kas_GetExecutionLevel(void)
 {
-    return __get_cpu_var(kasExecutionLevel);
+    unsigned long orig_level;
+
+    orig_level = get_cpu_var(kasExecutionLevel);
+    put_cpu_var(kasExecutionLevel);
+
+    return orig_level;
 }
 
 /** \brief Type definition for kas_spin_lock() parameter */
